lib/veritrans/api.rb

```diff
@@ -59,6 +60,20 @@ def create_snap_token(options = {})
     alias_method :create_widget_token, :create_snap_token
     alias_method :create_snap_redirect_url, :create_snap_token
 
+    # Create Snap payment page and return snap token
+    def get_snap_token(options = {})
+      result = request_with_logging(:post, config.api_host.sub('//api.', '//app.') + "/snap/v1/transactions", options)
+      Veritrans::SnapResult.new(result.response, result.url, result.request_options, result.time)
+      result.token
+    end
+
+    # Create Snap URL payment
+    def get_snap_url(options = {})
+      result = request_with_logging(:post, config.api_host.sub('//api.', '//app.') + "/snap/v1/transactions", options)
+      Veritrans::SnapResult.new(result.response, result.url, result.request_options, result.time)
+      result.redirect_url
+    end
```

Comment:
`get` is not the correct name, because `get` is only for retrieving. The function create & retrieve, so the name should be `create_`